error_init_auto.nit:34,5--9: Error: arity mismatch; prototype is 'init(x: Int)'.
error_init_auto.nit:36,5--15: Error: arity mismatch; prototype is 'init(x: Int)'.
error_init_auto.nit:37,5--18: Error: arity mismatch; prototype is 'init(x: Int)'.
error_init_auto.nit:38,5--16: Error: Method 'foo' doesn't exists in A.
